JsonClient/JsonClient/Program.cs

```diff
@@ -154,9 +169,9 @@ public static TObject BuildObjectByConsole<TObject>(TObject obj)
         public static object ParseValueToTheCorrect(Type pit, string val){
 
             if (pit == typeof(int))
-                return Int32.Parse(val);
+                return int.TryParse(val,out int res);
             else if (pit == typeof(decimal))
-                return Decimal.Parse(val);
+                return Decimal.TryParse(val, out Decimal res);
             else

```

Comment:
TryParse method always returns a `bool` object and you need either an int, decimal or string.